backend/pom.xml

```diff
@@ -62,6 +62,16 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.springfox</groupId>
```

Comment:
Az eredeti task leirasban levo (https://springdoc.org/#getting-started) dependencyvel mi volt a gond?
Elm ossz vissz ennyit kene hozza adni vele a depndencykhez, esetleg configba atallitan ia default endpointot, es az legalabb open-api 3-on alapul nem 2-n.:
   <dependency>
      <groupId>org.springdoc</groupId>
      <artifactId>springdoc-openapi-ui</artifactId>
      <version>1.5.9</version>
   </dependency>

Reply:
Az a baj vele, hogy nem a latest version, ha a 3.0.0 elérhető, és most szándékozunk bevezetni, akkor legyen ez. A UI-t támogatom én is, rögtön tesztelhető vele.